### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/gnucash/website/) for GnuCash/Website.

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -986,7 +986,7 @@ msgstr "Nicht offizielle Dokumentation"
 
 #: docs.phtml:698
 msgid "Some people have written HOWTO guides or tutorials."
-msgstr "Einige Nutzer haben HOWTOs oder Tutorials geschrieben:"
+msgstr "Einige Nutzer haben HOWTOs oder Tutorials geschrieben."
```
Comment (fellen) [inline po/de.po:989]: Ich glaube, hier muß ich eher den Quelltext ändern, da eine Liste folgt.

File: po/de.po [hunk 2]
```diff
@@ -1360,7 +1360,7 @@ msgstr "Zeige alle herunterladbaren GnuCash Dateien"
 
 #: externals/footer.phtml:12
 msgid "Server &amp; email outage reports to:"
-msgstr "Anfragen zu Server-Problemem an:"
+msgstr "Server- und E-Mail-Störung melden Sie an:"
```
Comment (fellen) [inline po/de.po:1363]: Auch hier sollte ich den Quelltext ändern. Eine Email bei Mailserver-Problemen ist wenig zielführend.

File: po/de.po [hunk 3]
```diff
@@ -1388,7 +1388,7 @@ msgstr "RSS abonnieren"
 
 #: externals/header.phtml:4
 msgid "Support the GnuCash project with a donation"
-msgstr "Das GnuCash Projekt mit einer Spende unterstützen."
+msgstr "Das GnuCash Projekt mit einer Spende unterstützen"
```
Comment (fellen) [inline po/de.po:1391]: It is a general question: Should tooltips of buttons be full sentences or should we probably apply the ellipses rule?

File: po/de.po [hunk 4]
```diff
@@ -1623,11 +1623,11 @@ msgstr ""
 msgid ""
 "A <b>summary bar</b> that displays all of the relevant account's balance "
 "information"
-msgstr "Eine <b>Summenzeile</b>, die alle relevanten Kontenstände auflistet."
+msgstr "Eine <b>Summenzeile</b>, die alle relevanten Kontenstände auflistet"
```
Comment (fellen) [inline po/de.po:1626]: Source file: Each element of the whole list should end with ";"

File: po/de.po [hunk 5]
```diff
@@ -1623,11 +1623,11 @@ msgstr ""
...
 
 #: features.phtml:59
 msgid "Many options to <b>customize</b> the appearance and the display style"
-msgstr "Viele Optionen um Aussehen und Inhalte anzupassen."
+msgstr "Viele Optionen um Aussehen und Inhalte anzupassen"
```
Comment (fellen) [inline po/de.po:1630]: Source file: … but the last element a full stop.

File: po/de.po [hunk 6]
```diff
@@ -114,8 +114,8 @@ msgstr ""
 #: 2.6-release-tour.phtml:60
 msgid "You can now attach a file or a location to any transaction."
 msgstr ""
-"Sie können jetzt an jede Transaktion eine Datei oder einen Speicherort "
-"anhängen."
+"Sie können jetzt an jede Transaktion eine Datei oder die Adresse für einen "
+"Speicherort anhängen."
```
Comment (fellen) [inline po/de.po:118]: Source: I think we should replace "attach" by "link".

**Review by fellen — COMMENTED** [review]
@CWehli Congrats, you found many errors in the sources!

File: po/nb.po [hunk 7]
```diff
@@ -125,13 +124,12 @@ msgid "Open datafile read-only"
 msgstr "Åpne datafil skrivebeskyttet"
 
 #: 2.6-release-tour.phtml:87
-#, fuzzy
 msgid ""
 "When opening a locked \"book\" one now has the option of opening it read-"
 "only."
 msgstr ""
 "Ved åpning av låst \"bokføring\" har man nå muligheten til å åpne den "
```
Comment (fellen) [inline po/nb.po:131]: @comradekingu Sorry for the intrusion. When I went through multiple-errors to search for source file weaknesses, I thought I append the full stop. But now I doubt "bokføring" is the right term here.

Comment (comradekingu) [reply po/nb.po:131]: Depends what "book" is supposed to mean. Translates to ~"bookkeeping"

Comment (fellen) [reply po/nb.po:131]: In this case "a file containing one book" is meant.

File: po/de.po [hunk 8]
```diff
@@ -2375,6 +2375,8 @@ msgid ""
 "Account matcher will decline to match accounts with a different commodity "
 "from the imported split if the import information includes the commodity."
 msgstr ""
+"Der Kontenabgleicher lehnt es ab, Konten mit einer anderen Ware als der "
+"importierten abzugleichen, wenn die Importinformationen die Ware enthalten."
```
Comment (fellen) [inline po/de.po:2379]: Commodity (Handelsgut) means currency (Währung) or security (Wertpapier). Suggestions for glossary?

**Comment by fellen** [comment]
Thanks to all!
As you can see, I fixed a few source strings. The result are 7 untranlated strings in de.po. Other languages behave similar.

File: po/de.po (hunk 3 above)
Comment (CWehli) [reply po/de.po:1392]: I think the tooltips are mainly to be understood as short hints and do not need punctuation marks. Unless two or more sentences are written.
The main thing is to remain consistent.

File: po/de.po [hunk 9]
```diff
@@ -2405,6 +2408,12 @@ msgid ""
 "<b>N.B.</b> This is incompatible with previous versions of the report and "
 "will require regenerating saved configurations."
 msgstr ""
+"Die Unterstützung für die britische Mehrwertsteuer und die australische GST "
+"wurde dem Bericht \"Einkommen-GST\" hinzugefügt. Die Optionen des Berichts "
+"werden von Quellkonten zu Quellverkaufs- und Einkaufskonten geändert, um "
+"eine ordnungsgemäße Meldung von Kapitalkäufen zu ermöglichen. <b> N.B.</b> "
+"Dies ist mit früheren Versionen des Berichts nicht kompatibel und erfordert "
+"eine Neugenerierung der gespeicherten Konfiguration."
```
Comment (CWehli) [inline po/de.po:2416]: How well known is the abbreviation N.B.?
According to the relevant reference books, it stands for "nota bene (lt.)" which means "note well" or "take notice".

**Pull request merged**